tests/test_pdf_download.py

```diff
@@ -141,36 +141,35 @@ class TestGetParseMeetingDate:
     the anchor tags returned by the get_meeting_links() function
     """
 
-    def test_standard_date_format(self):
-        """Tests the parsing of the standard date format in the anchor tags
-
-        TEST DATA
-        - A list of anchor tag that each contains the meeting date
-          in a standard format
+    @pytest.mark.parametrize(
+        'word_a,word_b,expected_distance',
+        [('possible', 'impossible', 2),
```

Comment:
Cool. We're giving a small subset of Levenshtein distance inputs and correct outputs here.